group/templates/group/editGroup.html

```diff
@@ -0,0 +1,88 @@
+copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+{% extends "index/base.html" %} 
+{% load static %}
+{% load bootstrap %}
+{% block title_name %}
+
+{% endblock title_name %}
+ {% block body_block %}
+<br>
+<br>
+<div class="container">
+  <a onclick="history.back()" type="reset" class="btn btn-sm btn-default">
+    Cancel
+  </a>
+  <br>
+  <br>
+  <div class="alert alert-dismissable alert-warning">
+    <button type="button" class="close" data-dismiss="alert">×</button>
+    <h4>Hey! You Now Setting for Group...</h4>
+    <p style="color:white">hint: HOLD shift to select more than two object</p>
+    <p style="color:white">hint: any change you made will save to the group</p>
+  </div>
+  <form action="." method="POST">
+    {% csrf_token %}
+    {% for field in form %}
+    {% if field.label_tag == form.gname.label_tag %}
+    <div class="panel panel-default">
+      <div class="panel-heading" style="font-size:16px">set group name</div>
+      <div class="panel-body">{{ field|bootstrap }}</div>
+    </div>
+    {% elif field.label_tag == form.owner.label_tag %}
+    <div class="panel panel-default">
+      <div class="panel-heading" style="font-size:16px">set owner</div>
+      <div class="panel-body">{{ field|bootstrap  }}</div>
+    </div>
+    {% elif field.label_tag == form.coowner.label_tag %}
+    <div class="panel panel-default">
+      <div class="panel-heading" style="font-size:16px">set coowner</div>
+      <div class="panel-body">{{ field|bootstrap  }}</div>
+    </div>
+    {% elif field.label_tag == form.member.label_tag %}
+    <div class="panel panel-default">
+      <div class="panel-heading" style="font-size:16px">set member</div>
+      <div class="panel-body">{{ field|bootstrap  }}</div>
+    </div>
+    {% elif field.label_tag == form.description.label_tag %}
+    <div class="panel panel-default">
+      <div class="panel-heading" style="font-size:16px">set description</div>
+      <div class="panel-body">{{ field|bootstrap  }}</div>
+    </div>
+    {% elif field.label_tag == form.announce.label_tag %}
+    <div class="panel panel-default">
+      <div class="panel-heading" style="font-size:16px">set announce</div>
+      <div class="panel-body">{{ field|bootstrap  }}</div>
+    </div>
+    {% elif field.label_tag == form.trace_contest.label_tag %}
+    <div class="panel panel-default">
+      <div class="panel-heading" style="font-size:16px">set trace contest</div>
+      <div class="panel-body">{{ field|bootstrap  }}</div>
```

Comment:
I'll note this and keep study on this.
But it seems less important then others unfinished things, I will finish others first.
Thanks for advise!